recipes/wagyu/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "wagyu" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0f0e4b78913e64083ca865f07e6aee30a24988890fc383f3003456287ed45d7f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - pybind11
+    - {{ compiler('cxx') }}
+    - conda-verify
+  host:
+    - python
+    - pip
+    - pybind11
+  run:
+    - python
+
+test:
+  imports:
+    - wagyu
+
+about:
+  home: https://github.com/lycantropos/wagyu
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
I've opened a PR upstream to try to get the license file included in the source distribution (https://github.com/lycantropos/wagyu/pull/1). If you could add the following comment in the `meta.yaml`:

```yaml
# License file manually packaged. See: https://github.com/lycantropos/wagyu/pull/1
license_file: LICENSE.txt
...
```

This will be a visual reminder to check if the license file is in the source distribution in future releases when you update the feedstock. If it is, then you can remove the comment and the manually packaged copy of the license.

Also note that upstream the license file is `LICENSE` not `LICENSE.txt`.